openpecha/corpus/download.py

```diff
@@ -0,0 +1,41 @@
+import os
+from pathlib import Path
+
+import requests
+
+from openpecha.github_utils import get_github_repo
+
+
+def get_base_vol_list(pecha_repo, pecha_id):
+    base_vols = []
+    contents = pecha_repo.get_contents(f"./{pecha_id}.opf/base")
+    for content_file in contents:
+        base_vols.append(content_file.name)
+    return base_vols
+
+
+def download_base_vols(output_path, pecha_id, base_vols):
+    for base_vol in base_vols:
+        base_response = requests.get(
+            f"https://github.com/OpenPecha/{pecha_id}/blob/master/{pecha_id}.opf/base/{base_vol}"
```

Comment:
this url will download the page not the base text, instead use this url https://raw.githubusercontent.com/OpenPecha/P000001/master/P000001.opf/base/v001.txt